styles/new/base.css

```diff
@@ -3,9 +3,9 @@
 }
 
 code {
-  @apply rounded-[4px]
+  @apply rounded
```

Comment:
Is.. this change really necessary? 🤔

Reply:
If there is a class provided by Tailwind, I think it should be managed with this class.
  - Allows editing and formatting styles in a one place
  - [Tailwind uses rem](https://github.com/tailwindlabs/tailwindcss/blob/master/stubs/config.full.js#L87-L96) for border radius, it would be annoying if this area remains fixed while the rest of the page is updated relative to the browser font size.

So, should this situation be resolved under this PR?

I think not 😄  I accidentally created a class for 4px (0.25rem) and while checking the files I noticed that it was also used in this file. Later, I saw that this class was unnecessary and changed it to `rounded`. I can update it back here if you want

Reply:
Fine by having it here.